yarn-project/aztec-rpc/src/database/note_spending_info_dao.ts

```diff
@@ -16,10 +16,6 @@ export interface NoteSpendingInfoDao {
    * The nonce of the note.
    */
   nonce: Fr;
-  /**
-   * The owner of the note.
-   */
-  ownerAddress: AztecAddress;
```

Comment:
I don't really follow this change, we only recently added this I think. Since the introduction of account abstraction, isn't the owner of every note an Aztec Address? If I send funds to someone, I send them to their address. Likewise, when a contract wants to spend funds it should be spending funds associated with an address.

Reply:
I think it’s that a note doesn’t technically need an ‘owner’. A note can be customised to be any struct of information, and an ‘owner’ might not be needed.
I don’t think the notion of ownership can be automatically established for every note. It depends on the contract’s logic. So it’s up to each dapp to determine what ‘ownership’ means for a particular storage slot.
As a not-great-but-quickly-thought-up example, imagine a contract which tries to emulate a traditional bank. A note might be encrypted for both the recipient and the banker. Maybe only the bank can execute the transfer function on behalf of users. The notion of an ‘owner’ here could mean two things: is it the customer whose balance is being modified, or is it the banker who has permission to modify the state (by producing a nullifier)? 
Or imagine a note for which a collection of people may produce a nullifier to modify it. Who’s the owner? Multiple people. Permission to modify the note might be based on some contract check comparing msg_sender against a whitelist. It’s too complicated for the generic note db to determine an ‘owner’.
Or imagine a note containing public information. Something like the defi interaction note from Aztec Connect. It has no owner: it’s just some public data that can be referenced (read) by multiple users, from a private function.
Or maybe some shared game state, where the game is private to the outside world, but all participants have been sent information about the game state (stored as a note), but the person who may next edit the game state depends on some complex game logic.